tests/metrics_test.go

```diff
@@ -153,14 +157,99 @@ var _ = Describe("Metrics", func() {
 		)
 	})
 
+	Context("SSP metrics", func() {
+		var originalVersion string
+
+		getDeployment := func(name string, namespace string) *apps.Deployment {
```

Comment:
This can be a free function, because it does not depend on any variables inside the scope.

Reply:
moved